content/docs/0.19.x/_index.md

```diff
@@ -5,6 +5,7 @@ weight: 977
 sidebar_multicard: true
 icon: docs
 hide: true
+develop: This documentation is currently under development and officially released with the next Keptn release. 
```

Comment:
```suggestion
develop: This documentation is currently under development for a future Keptn release. 
```
We can't guarantee that this will be the next release.  We have hide: true set so customers shouldn't see this anyhow.

Reply:
Fixed it. Thx!